test/testdata/definition_validator/superclass_class_loc__2.rbi

```diff
@@ -0,0 +1,18 @@
+# typed: true
+
+# This is a rather large comment at the start of the file which exposes a bug
+# where we were accidentally turning a LocOffset from this file into a Loc for
+# the other file. With a sufficiently long comment, that causes an array access
+# out of bounds.
+# This is a rather large comment at the start of the file which exposes a bug
+# where we were accidentally turning a LocOffset from this file into a Loc for
+# the other file. With a sufficiently long comment, that causes an array access
+# out of bounds.
+# This is a rather large comment at the start of the file which exposes a bug
+# where we were accidentally turning a LocOffset from this file into a Loc for
+# the other file. With a sufficiently long comment, that causes an array access
+# out of bounds.
```

Comment:
Note that this is separate from the fast path—this causes a crash even in the slow path.